lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -1,23 +1,55 @@
 /*
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
+ * 
  */
 import { BrowserStringUtils } from "../utils/BrowserStringUtils";
 import { BrowserAuthError } from "../error/BrowserAuthError";
-
+import { StringUtils } from "@azure/msal-common";
+/**
+ * See here for more info on RsaHashedKeyGenParams: https://developer.mozilla.org/en-US/docs/Web/API/RsaHashedKeyGenParams
+ */
+// RSA KeyGen Algorithm
+const PKCS1_V15_KEYGEN_ALG = "RSASSA-PKCS1-v1_5";
 // SHA-256 hashing algorithm
-const HASH_ALG = "SHA-256";
+const S256_HASH_ALG = "SHA-256";
```

Comment:
I think `HASH_ALG` is a better name. I personally don't think constants like this need to include hints about their value.

Reply:
One reason I did this is because without this we do not know what this constant is referring to without looking at the value. This improves readability and makes dev a little easier.

Reply:
My 2c: I would also prefer to be more precise with the naming. Updating this here makes it more readable. Same thing below.